File: app/src/main/java/com/example/artikproject/background_work/service/MyWorker.java
package com.example.artikproject.background_work.service;

import android.content.Context;
import android.os.Build;

import androidx.annotation.NonNull;
import androidx.annotation.RequiresApi;
import androidx.work.Worker;
import androidx.work.WorkerParameters;

public class MyWorker extends Worker {

    public MyWorker(
            @NonNull Context context,
            @NonNull WorkerParameters params) {
        super(context, params);
    }

    @NonNull
    @RequiresApi(api = Build.VERSION_CODES.O)
    @Override
    public Result doWork() {
        new PlayService().get_group_db(getApplicationContext());
        return Result.success();
    }
}

File: app/src/main/java/ru/agpu/artikproject/background_work/service/MyWorker.kt
package ru.agpu.artikproject.background_work.service

import android.content.Context
import android.os.Build
import androidx.annotation.RequiresApi
import androidx.work.Worker
import androidx.work.WorkerParameters

class MyWorker(context: Context, params: WorkerParameters): Worker(context, params) {
    @RequiresApi(api = Build.VERSION_CODES.O)
    override fun doWork(): Result {
        CheckRaspChanges(applicationContext)
        return Result.success()
    }
}